lib/runner.js

```diff
@@ -130,11 +158,45 @@ class Runner {
         process.send({
             event: 'runner:end',
             failures: failures,
-            cid: this.cid
+            cid: this.cid,
+            specs: this.specs
         })
         process.exit(failures === 0 ? 0 : 1)
     }
 
+    addTestDetails (payload) {
+        if (this.currentTest) {
+            payload.title = this.currentTest.title
+            payload.parent = this.currentTest.parent
```

Comment:
After I finished the [cucumber integration](https://github.com/webdriverio/wdio-cucumber-framework) and revisit the other framework integration we actually always send the title and parent of a test spec. So never mind the comment above.